kafka-ui-react-app/src/widgets/ClusterConfigForm/ClusterConfigForm.styled.ts

```diff
@@ -60,3 +60,24 @@ export const FileUploadInputWrapper = styled.div`
   align-items: center;
   color: ${({ theme }) => theme.clusterConfigForm.fileInput.color}};
 `;
+
+export const Spinner = styled.div(
```

Comment:
i think maybe we can just make a general one cause i see this spinner code thing repeated across the application. we can make it customizable and just use that with `height` and `widths` and `colors` `props` etc and set the most common used one as a default value.

Reply:
Yeah, I thought about it too. Update PR soon, thanks!

Reply:
@Mgrdich @workshur updated spinner